content/docs/concepts/multicluster-deployments/index.md

```diff
@@ -12,39 +12,94 @@ a single controllable composite application.
 
 For applications up to a certain size, all of the microservices comprising the
 application can be running on a single orchestration platform (e.g., Kubernetes cluster).
-However, for many reasons such as scale, redundancy, compliance, etc., most applications
-will eventually need to be distributed and have some of their services running elsewhere.
+Eventually, you may need to distribute most applications and run some or all of their services elsewhere due to one or more of the following reasons:
+
+- Scale
+- Redundancy
+- Workload isolation
+- Compliance
+- Integration with legacy services
 
 Istio supports many possible topologies for distributing the services of an
 application beyond a single cluster, for example:
 
-* Services within the mesh can use [service entries](/docs/concepts/traffic-management/#service-entries)
+- You can use [egress traffic control](/docs/tasks/traffic-management/egress/) to allow workloads inside the mesh
   to access standalone external services or to access services exposed by another loosely-coupled service mesh.
-* You can [expand the service mesh](/docs/setup/kubernetes/additional-setup/mesh-expansion/) to include
+- You can [expand the service mesh](/docs/setup/kubernetes/additional-setup/mesh-expansion/) to include
   services running on VMs or bare metal hosts.
-* You can combine the services from more than one cluster into a single composite service mesh.
+- You can combine the services from more than one cluster into a single composite service mesh.
+- You can combine autonomous service meshes in multiple clusters into a loosely coupled [federation of service meshes](#mesh-federation).
+
+## Single multicluster service mesh or mesh federation?
+
+As you plan your multicluster topology, you need to determine if you should combine the clusters into
+[a single multicluster mesh](#multicluster-service-mesh) or create [a federation of multiple meshes](#mesh-federation).
+To make that decision, ask yourself the following questions:
+
+1. Do you enforce **{{< gloss >}}uniform naming{{< /gloss >}}** in your clusters?
+
+1. Do you want your combined clusters to **expose** all their services to each other? Due to compliance or security
```

Comment:
With the new Sidecar API it is possible to restrict service exposure using namespaces, rather than meshes. So this may no longer be a decision point for whether you need multiple meshes.

Reply:
I do not think Sidecar API can serve as a security measure. The security requirement, I guess, is that all the incoming traffic to the cluster is denied by default. You have to specify explicitly which services you expose. With Sidecar API - everything is exposed by default.

Reply:
That seems like the same question as the control all traffic that enters question then?

All I'm pointing out is that it isn't a requirement of a mesh that all services are exposed to each other, that is just the default (and a default we can support changing).

The primary issue is actually more about namespace vs. cluster isolation. If namespace isolation is good enough, a single mesh will work, if you need cluster isolation, you need multiple meshes.